### PR TITLE
Add Tukey's range test

### PR DESCRIPTION
**Pull request by kotarou3**

### CODE REVIEW

**Comment by trevnorris** [comment]
Landed in 7d957e83dac034f52c2aa433384c7f6a293ce09a and 827c5434782ab4826a4a41d212882527ba51b62b. Thanks!

File: dist/jstat.js [hunk 1]
```diff
@@ -2799,6 +2810,363 @@ jStat.extend(jStat.laplace, {
...
+     / (((( yi * q4 + q3) * yi + q2) * yi + q1) * yi + q0);
+  if (v < vmax) t += (t * t * t + t) / v / 4.0;
+  var q = c1 - c2 * t;
+  if (v < vmax) q += -c3 / v + c4 * t / v;
+  return t * (q * Math.log(c - 1.0) + c5);
+}
+
+jStat.extend(jStat.tukey, {
+  cdf: function cdf(q, nmeans, df) {
+    // Identical implementation as the R ptukey() function as of commit 68947
```
Comment (kgryte) [inline dist/jstat.js]: If this implementation was lifted from R, @trevnorris, then you should consider that this code is under a GPL license. Similar for other functions included in this PR.

**Comment by trevnorris** [comment]
Released in [v1.6.0](https://github.com/jstat/jstat/releases/tag/1.6.0)

File: dist/jstat.js (hunk 1 above)
Comment (trevnorris) [inline dist/jstat.js]: Thanks for pointing that out. IANAL so not sure there that line is drawn. I'll look at the source in R and see if there are similar implementations out there that aren't GPL.

**Pull request closed without merging**